api/CSSMediaRule.json

```
@@ -3,6 +3,10 @@
     "CSSMediaRule": {
       "__compat": {
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSSMediaRule",
+        "spec_url": [
+          "https://drafts.csswg.org/css-conditional/#the-cssmediarule-interface",
+          "https://drafts.csswg.org/cssom/#the-cssmediarule-interface"
+        ],
```

The CSSOM spec refers to CSS conditional. No need for it to be here imo.

```suggestion
        "spec_url": "https://drafts.csswg.org/css-conditional/#the-cssmediarule-interface",
```